scripts/upload.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+# REQUIREMENTS
+#   fswatch
+#
+# HOW TO USE
+# 1. Make this file executable with the following:
+#   `chmod 700 upload.sh`
+# 2. Add the following function to your shell (bash or zsh):
+#   `function crank_shaft { fswatch -0 -1 . | xargs -0 -I $ ./upload.sh }`
+# 3. In the theme directory, run `crank_shaft`
+# 4. After about 30 seconds, edits saved in local files will appear on the site
```

Comment:
It seems to take at most 30 seconds, I would say the average is closer to 10 to 15. It does vary with internet speed though

Reply:
OK, updated.